pkg/tests/ossm/smoke_test.go

```diff
@@ -96,13 +96,23 @@ func TestSmoke(t *testing.T) {
 			assertTrafficFlowsThroughProxy(t, ns)
 		})
 
-		// This test case is flaky and disabled for now: https://issues.redhat.com/browse/OSSM-4064
-		// t.NewSubTest(fmt.Sprintf("delete smcp %s", toVersion)).Run(func(t TestHelper) {
-		// 	t.Logf("This test checks whether SMCP %s deletion delete all the resources", env.GetSMCPVersion())
-
-		// 	t.LogStep("Delete SMCP and verify if this deletes all resources")
-		// 	assertUninstallDeletesAllResources(t, env.GetSMCPVersion())
-		// })
+		t.NewSubTest(fmt.Sprintf("delete smcp %s", toVersion)).Run(func(t TestHelper) {
+			t.Logf("This test checks whether SMCP %s deletion delete all the resources", env.GetSMCPVersion())
+
+			t.LogStep("Delete SMCP and verify if this deletes all resources")
+			t.LogStep("Delete SMCP in namespace " + meshNamespace)
```

Comment:
A step usually contains one or more actions. The first step doesn't, so the log output looks weird (step 1 and then immediately step 2).

Reply:
Also, you're deleting both the SMCP and SMMR in this step, so the logstep is misleading.

Reply:
Another suggestion is to change "This test checks whether SMCP %s deletion deletes all the resources"

Reply:
Done